release-notes/opensearch-release-notes-1.2.1.md

```diff
@@ -0,0 +1,5 @@
+# OpenSearch 1.2.1 Release Notes
```

Comment:
Is it just opensearch or dashboard too?

Reply:
OpenSearch Dashboards was _not_ released for 1.2.1 as it was unaffected

Reply:
Looking at previous release notes https://github.com/opensearch-project/opensearch-build/blob/main/release-notes/opensearch-release-notes-1.2.0.md